File: 01.JAVA/pg-166-169/pg_166_05.java
import java.util.Scanner;
public static void main(String[] args)
{
    Scanner enter = new Scanner(System.in);
    int dig1, dig2;

    System.out.println("");
    System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
    System.out.println("READ A TWO-DIGIT INTEGER AND DETERMINE IF BOTH ARE EVEN");
    System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVV");
    System.out.println("");

    System.out.print("Write a two digit integer: ");
    int num = enter.nextInt();

    if(num < 0)
    {
        num *= -1;
    }
    
    if(num > 0)
    {
        dig2 = num % 10;
        dig1 = num / 10;

        if(dig1 % 2 == 0 && dig2 % 2 == 0)
        {
            System.out.println("Both digits are even.");
        }
        else
        {
            System.out.println("Both digits are not even.");
        }
    }
}

File: 06.PHP/pg-166-169/pg_166_05.php
<?php
echo("");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPPHPHPHPHPHPHPHPHPHPHPHPHPH<br>");
echo("READ A TOW-DIGITS INTEGER AND DETERMINE IF BOTH DIGITS ARE EVEN<br>");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPPHPHPHPHPHPHPHPHPHPHPHPHPH<br><br>");
echo("");

$num = 85   ;
$dig1 = 0;
$dig2 = 0;

echo($num . "<br>");

if($num < 0)
    {
        $num *= -1;
    }

if($num >= 10 & $num <=99)
    {
        $dig2 = $num % 10;
        $dig1 = intdiv($num, 10);

        if($dig1 % 2 == 0 & $dig2 % 2 == 0)
            {
                echo("Both digits are even.");
            }
            else
                {
                    echo("Both digits are not even.");
                }
    }
    else
        {
            echo("The written number doesn't have two digits. Please try again!");
        }
?>